libs/blocks/aside/aside.css

```diff
@@ -634,18 +635,18 @@
     min-height: 560px;
   }
 
-  .aside.large {
-    min-height: 700px;
-  }
 
```

Comment:
Yea that's a good point. The reason this was removed was because the spectrum mockup specifies these new aside large blocks to be 640px in height instead of 700px. Removing this allows the height to fall to the height of the blocks content. However, we could consider explicitly setting a min-height of 640px here instead, which might help mitigate the impact on existing production use cases.

Reply:
Yep, setting it to `640` is likely the better definition to ensure current marquees aren't impacted too much.